.github/workflows/main.yml

```diff
@@ -114,75 +147,49 @@ jobs:
           key: hdf5
           path: ${{github.workspace}}/hdf5install
           
-      - name: Build REST VOL (Autotools)
+      - name: Clone vol-tests repo
+        uses: actions/checkout@v3
+        with:
+          path: ${{github.workspace}}/rest-vol/test/vol-tests/
+          repository: mattjala/vol-tests
```

Comment:
Are there still specific fixes in your fork of the VOL tests to make the test run correctly? We should probably base this workflow off of the upstream so that it's useful to see the state of the current VOL tests